### PR TITLE
Apply latest token ds-auro-icon-size and adjust styles applied to icon #55

### PR DESCRIPTION
**Pull request by rmenner**
# Alaska Airlines Pull Request

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Apply the latest 'ds-auro-icon-size' token to icon styles and adjust the styling logic for icons in the AuroNav component to enhance consistency and maintainability.

New Features:
- Apply the latest token 'ds-auro-icon-size' to icon styles.

Enhancements:
- Adjust icon styles to use the 'ds-auro-icon-size' token for consistency.

### CODE REVIEW

File: src/auro-nav.js [hunk 1]
```diff
@@ -235,11 +235,8 @@ export class AuroNav extends LitElement {
     icon.category = category;
     icon.name = name;
 
-    icon.style.width = size;
-    icon.style.height = size;
-    icon.style.lineHeight = `1.5rem`;
+    icon.style = size;
```
Comment (sun-mota) [inline src/auro-nav.js:238]: The param name `size` seems misleading as it can be any style-object.

File: src/auro-nav.js [hunk 2]
```diff
@@ -235,11 +235,8 @@ export class AuroNav extends LitElement {
     icon.category = category;
```
Comment (sun-mota) [inline src/auro-nav.js:235]: line:232, `customSize` attribute is deprecated, we can remove this line

Comment (rmenner) [reply src/auro-nav.js:234]: fixed this

File: src/auro-nav.js (hunk 1 above)
Comment (rmenner) [reply src/auro-nav.js]: updated

**Review by jordanjones243 — APPROVED** [review]

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
Need to rebase this repo off of main

**Pull request merged**